app/components/LiveTransactionStatsCard.tsx

```diff
@@ -266,6 +266,7 @@ function AnimatedTransactionCount({ info }: { info: PerformanceInfo }) {
     );
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 function PingStatsCardBody({ series, setSeries }: { series: Series; setSeries: SetSeries }) {
     const pingInfo = useSolanaPingInfo();
```

Comment:
https://github.com/nitro-svm/explorer/blob/3380a613bd69df594f43d87274fa9a0037ed9e60/app/providers/stats/SolanaPingProvider.tsx#L12-L20

If we want to support ping data, we'll need to update this endpoint to hit our own healthcheck.